tests/10apidoc/33room-members.pl

```diff
@@ -178,10 +172,14 @@ sub matrix_join_room
          assert_json_keys( $body, qw( room_id ) );
          assert_eq( $body->{room_id}, $room_id );
 
-         matrix_get_room_state( $user, $room_id,
-            type      => "m.room.member",
-            state_key => $user->user_id,
-         )
+         # Retry getting the state a few times, for it may take some time to
+         # propagate in a multi-process homeserver
+         retry_until_success {
+            matrix_get_room_state( $user, $room_id,
+               type      => "m.room.member",
+               state_key => $user->user_id,
+            )
+         }
       })->then( sub {
```

Comment:
should this not be inside the `retry_until_success` call?

Reply:
(likewise various other places below)

Reply:
The whole subroutine?